diff --git a/tools/gyp/pylib/gyp/generator/ninja.py b/tools/gyp/pylib/gyp/generator/ninja.py
index ae3dded..7ba9483 100644
--- a/tools/gyp/pylib/gyp/generator/ninja.py
+++ b/tools/gyp/pylib/gyp/generator/ninja.py
@@ -1466,12 +1466,13 @@ class NinjaWriter:
             # flags can still override them.
             ldflags = env_ldflags + config.get("ldflags", [])
             if is_executable and len(solibs):
-                rpath = "lib/"
-                if self.toolset != "target":
-                    rpath += self.toolset
-                    ldflags.append(r"-Wl,-rpath=\$$ORIGIN/%s" % rpath)
-                else:
-                    ldflags.append("-Wl,-rpath=%s" % self.target_rpath)
+                rpath = ""
+                # if self.toolset != "target":
+                #     rpath += self.toolset
+                #     ldflags.append(r"-Wl,-rpath=\$$ORIGIN/%s" % rpath)
+                # else:
+                #     ldflags.append("-Wl,-rpath=%s" % self.target_rpath)
+                ldflags.append(r"-Wl,-rpath=\$$ORIGIN/%s" % rpath)
                 ldflags.append("-Wl,-rpath-link=%s" % rpath)
         self.WriteVariableList(ninja_file, "ldflags", map(self.ExpandSpecial, ldflags))
 
