diff --git a/tools/v8_gypfiles/v8.gyp b/tools/v8_gypfiles/v8.gyp
index 0914746..f7c89f0 100644
--- a/tools/v8_gypfiles/v8.gyp
+++ b/tools/v8_gypfiles/v8.gyp
@@ -1609,12 +1609,12 @@
         '<(V8_ROOT)/src/libplatform/worker-thread.cc',
         '<(V8_ROOT)/src/libplatform/worker-thread.h',
       ],
+      'direct_dependent_settings': {
+        'defines': ['USING_V8_PLATFORM_SHARED'],
+      },
+      'defines': ['BUILDING_V8_PLATFORM_SHARED'],
       'conditions': [
         ['component=="shared_library"', {
-          'direct_dependent_settings': {
-            'defines': ['USING_V8_PLATFORM_SHARED'],
-          },
-          'defines': ['BUILDING_V8_PLATFORM_SHARED'],
         }],
         ['v8_use_perfetto==1', {
           'sources!': [
